MIP86/MIP86.md

```diff
@@ -9,11 +9,12 @@ Author(s): @CoinSharesCo
 Contributors: @jmmognetti
 Tags: PSM
 Type: General
-Status: Formal Submission
+Status: Rejected
 Date Proposed: 2022-09-13
-Date Ratified:
+Date Ratified: 2022-11-28
```

Comment:
Erase date.

Reply:
Email

Reply:
Sorry I'm still wanting be involved I just need a mentor someone to guide me and show me how these things work I've looked at the guidelines and stuff for the maker forum there's a lack of communication please provide one on one training for those eager to learn but no trainees